### PR TITLE
country-codes to country-names

### LINKED ISSUE #12: country names
Display on the website the country names and not the sent by the API country codes

### PR DESCRIPTION
**Pull request by catalinac3**
closes #12 

- copied object with {countries codes: country names} from a Gists, added the Gist link on my Readme file.
- added extra file of js where the object is turned into a map and added the function codeCountryConversion 
- made an extra folder for js files. Reason for which, I changed the importing of js files to adapt to the new folder, and added an <script src= > to include the second js file.

### CODE REVIEW

**Review by keponk — APPROVED** [review]
this looks great! :) go ahead and merge.
My only 2 cents, is there a special reason you wanted to use "Map" as a data structure? normally we use it for specific use cases, otherwise a regular object would suffice.  If done with an object : `countryCode in mapCountries` does exactly the same as `has`.

awesome job!

**Comment by catalinac3** [comment]
There is no special reason, maybe "curiosity" ... My programming experience is in python and there, the structure is a dictionary, and I try finding the same in javascript. Then I notice the object - map difference and after reading a bit, I got curious. So I did it to learn how map works in javascript. I will change it :)

**Comment by catalinac3** [comment]
btw, I thought maps were a data structure, how are maps used for js, or for what ?

**Comment by keponk** [comment]
> btw, I thought maps were a data structure, how are maps used for js, or for what ?

ah makes sense if you're coming from Python :) 
One cool thing about JS is that the "default"  object type can already be used as a dictionary!

When thinking about data structures, specially in the high level languages, focus on why you want to use it. The dictionary is perfect for this country list because you have a code (a key) that corresponds to a country (a value). There are actually many other properties of dictionary but our use case right now is just that.
You would then in python access it like `countryCode["ABC"]` 

Now let's see what this means in javascript. I need to create a variable that holds keys that correspond to specific values and I'd like to use or access them in more or less the same way. That is the exact behavior of the "default" javascript object!  ^^ 

This raises the question "then why do Map exists in javascript?" , and the reason is that like dictionaries, having access to a value by using a key is only the "main" property of that data structure, but there are other properties. In this case the "extra" thing Map does is that it "remembers" the order in which the keys where added to that Map. That's cool but we probably don't need or care about that property in our use case. 

In other words, you could continue to use Map and it will work exactly as you intended but from the developers side it would "look strange" to use a non-default data structure for something that the default one can do. 

I also found this same question in SO with a healthy discussion and comparison in case you are curious to know more! https://stackoverflow.com/questions/18541940/map-vs-object-in-javascript
